src/test/java/api/loans/CheckInByBarcodeTests.java

```diff
@@ -742,7 +745,7 @@ public void overdueRecallFineShouldBeChargedWhenItemIsOverdueAfterRecall() {
 
     JsonObject account = createdAccounts.get(0);
 
-    assertThat(account, OverdueFineMatcher.isValidOverdueFine(loan, nod,
+    assertThat(account, OverdueFineMatcher.isValidOverdueFine(checkedInLoan, nod,
```

Comment:
Personally, I might statically import `isValidOverdueFine` to reduce the noise in this assertion

Reply:
Done